pages/blocks/form/form.js

```diff
@@ -388,7 +388,7 @@ function createForm({
       formField += `
         <div class="input-el question ${required ? 'is-required' : ''}">
           <div class="title-el">
-            <label class="label-title" for="${name}">${setup.label}</label>
+            <label class="label-title" tabindex="0" for="${name}">${setup.label}</label>
```

Comment:
Why putting a tabindex on a label?

Reply:
(It isn't expected to be focusable)

Reply:
I recommend leaving all tabindex="0" out - this isn't recommended. It actually creates a different problem when users expect that a focusable item is interactive, but the legends are not. The assistive technology will use the semantic association to read the groups correctly without the legend be focusable.

Reply:
I've removed them.

Reply:
FYI, I added them yesterday and saw your comment today from ticket to add the fieldset/legends. Forgot to remove them. Good catch.